docs/src/main/asciidoc/kubernetes-guide.adoc

```diff
@@ -73,6 +73,8 @@ In this section, we are going to leverage the build mechanism of OpenShift. Run:
 # To build the image on OpenShift
 oc new-build --binary --name=quarkus-quickstart -l app=quarkus-quickstart
 oc patch bc/quarkus-quickstart -p '{"spec":{"strategy":{"dockerStrategy":{"dockerfilePath":"src/main/docker/Dockerfile.native"}}}}'
+# For Windows Users
+oc patch bc/quarkus-quickstart -p "{\"spec\":{\"strategy\":{\"dockerStrategy\":{\"dockerfilePath\":\"src/main/docker/Dockerfile.native\"}}}}"
```

Comment:
I think my preference would be to use the same quoting for both versions. The Windows quoting should work on Linux too. It's a bit less readable but probably better than having two versions, especially since people will copy/paste these things without reading them too much and then they would have executed a command twice.

@geoand WDYT?

Reply:
Changed to use only escaped version. I don't have Linux so need someone to verify it works.

Reply:
If the same string works, then let's use it. I will test it on Linux and let you know

Reply:
I tried it and it works just fine on Linux as well, so :+1:  from me